components/back.ftd

```diff
@@ -0,0 +1,51 @@
+-- import: fastn-community.github.io/midnight-business-card/components/lib
+
+-- component back:
+optional caption name:
+optional string company-name:
+optional string title:
+optional string phone-no:
+optional string email:
+optional string address:
+optional ftd.image-src logo:
+optional ftd.image-src avatar:
+optional string company-slogan:
+optional ftd.image-src qr-code:
+optional string website:
```

Comment:
Please refer the fields available in the card component in the business-card-template. Also remove all the unused fields.